src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/TextViewWindow_OutOfProc.Verifier.cs

```diff
@@ -58,15 +60,27 @@ public Verifier(TTextViewWindow textViewWindow, VisualStudioInstance instance)
                 {
                     if (ensureExpectedItemsAreOrdered)
                     {
-                        TestUtilities.ThrowIfExpectedItemNotFoundInOrder(
-                            actions,
-                            expectedItems);
+                        if (verifyCompleteList)
```

Comment:
if you want to make sure certain code action always stay at top like "add using" even if new code actions are added or engine changed. I think it would be better just checking its position rather than whole list which is too fragile to break as people adds more builtin analyzers/fixer/codeactions.

Reply:
➡️ Discussion is ongoing in https://github.com/dotnet/roslyn/pull/27280#discussion_r192843897.